draggableMotion/src/main/ets/components/MainPage/modelClass.ets

```diff
@@ -0,0 +1,134 @@
+/*
```

Comment:
Change the file name from modelClass.ets to DraggableModel.ets

Reply:
done